crda, once again, does not build with libressl

## Issue description
I am seeing a new (but similar to #53529) build problem with crda when using a global libressl override. Things were working fine until maybe a week ago, I think.

I am confused, because I can't see anything relevant that has changed lately, but for some reason something about `sk_deep_copy` has changed for libressl or m2crypto. Does anyone know what is going on?

```
unpacking sources
unpacking source archive /nix/store/mzd0b2idc5n0fjpmdp8qs81b9sfw6axq-crda-3.18.tar.xz
source root is crda-3.18
setting SOURCE_DATE_EPOCH to timestamp 1418255489 of file crda-3.18/utils/key2pub.py
patching sources
patching script interpreter paths in utils/
utils/key2pub.py: interpreter directive changed from "/usr/bin/env python" to "/nix/store/ghpwwhpniw0d3nphkd7hh5ljxvivgw5x-python-2.7.15/bin/python"
configuring
no configure script, doing nothing
building
build flags: -j32 -l32 SHELL=/nix/store/cinw572b38aln37glr0zb8lxwrgaffl4-bash-4.4-p23/bin/bash PREFIX=\$\(out\) SBINDIR=\$\(out\)/bin/ UDEV_RULE_DIR=\$\(out\)/lib/udev/rules.d/ REG_BIN=/nix/store/q70rnjkwnpd8p08jpmsjl6pzc1p1q6k0-wireless-regdb-2018.10.24/lib/crda/regulatory.bin all_noverify
  GEN  keys-gcrypt.c
  Trusted pubkeys: pubkeys/sforshee.key.pub.pem pubkeys/linville.key.pub.pem
ERROR: Failed to import the "M2Crypto" module: /nix/store/fpwqzw96gzvbjdzhvd7rlgz2v19p13i9-python2.7-M2Crypto-0.30.1/lib/python2.7/site-packages/M2Crypto/_m2crypto.so: undefined symbol: sk_deep_copy
Please install the "M2Crypto" Python module.
On Debian GNU/Linux the package is called "python-m2crypto".
make: *** [Makefile:115: keys-gcrypt.c] Error 1
builder for '/nix/store/2nfq5sxnmhd1bw07qq1d0nwxh53aiqni-crda-3.18.drv' failed with exit code 2
```

--- a/pkgs/development/python-modules/m2crypto/default.nix
+++ b/pkgs/development/python-modules/m2crypto/default.nix
@@ -25,6 +25,7 @@ buildPythonPackage rec {
   ];
   patchFlags = "-p0";
 
+  nativeBuildInputs = [ swig2 ];
   buildInputs = [ swig2 openssl ];
 
   propagatedBuildInputs = [ typing ];

